### ISSUE TITLE
Update to include changes in Sandbox

### ISSUE THREAD

**Issue by kateefly** [DLFMetadataAssessment/EnvironmentalScan#2]
Important changes include:
* Update config.yml so it doesn't have collections info
* Converting html pages to markdown pages
* Add html parser to top of markdown files

**Comment by kateefly** [comment]
More detail on commit 56f80ab85346468ea044da084d5fb2b0a953206c. I reviewed all changes that have happened to the Sandbox since the creation of this child project so that it could match what's currently in the Sandbox as closely as possible.

The following were updated to match Sandbox:
- Includes/Nav
- Update Gemfile.lock
- Config.yml
- Review.html
- Deleted _review folder
- Deleted page.html layout

Includes content from commits 137d1615b0990ea1f343cdeb2a4286fe95482c0a and 15a57310713963baa31096c2198ad7d386a13138. Does **not** include adding html parser code to top of md files or converting HTML files to md. This work will be taken care of by issue#5.
